docs/proposals/04-new-core-gardener-cloud-apis.md

```diff
@@ -0,0 +1,727 @@
+  conditions:
+  - lastTransitionTime: "2020-07-14T19:16:42Z"
+    lastUpdateTime: "2020-07-14T19:18:17Z"
+    message: all checks passed
+    reason: Passed
+    status: "True"
+    type: Available
+  gardener:
+    id: 4c9832b3823ee6784064877d3eb10c189fc26e98a1286c0d8a5bc82169ed702c
+    name: gardener-controller-manager-7fhn9ikan73n-7jhka
+    version: 1.0.0
+  observedGeneration: 1
+```
+
+### `SecretBinding` resource
+
+Special note: No modifications needed compared to the current `garden.sapcloud.io/v1beta1.SecretBinding` resource.
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secret1
+  namespace: garden-core
+type: Opaque
+data:
+  # <some-provider-specific data keys>
+  # https://github.com/gardener/gardener-extensions/blob/master/controllers/provider-alicloud/example/30-infrastructure.yaml#L14-L15
+  # https://github.com/gardener/gardener-extensions/blob/master/controllers/provider-aws/example/30-infrastructure.yaml#L9-L10
+  # https://github.com/gardener/gardener-extensions/blob/master/controllers/provider-azure/example/30-infrastructure.yaml#L14-L17
+  # https://github.com/gardener/gardener-extensions/blob/master/controllers/provider-gcp/example/30-infrastructure.yaml#L14
+  # https://github.com/gardener/gardener-extensions/blob/master/controllers/provider-openstack/example/30-infrastructure.yaml#L15-L18
+  # https://github.com/gardener/gardener-extensions/blob/master/controllers/provider-packet/example/30-infrastructure.yaml#L14-L15
+  #
+  # If you use your own domain (not the default domain of your landscape) then you have to add additional keys to this secret.
+  # The reason is that the DNS management is not part of the Gardener core code base but externalized, hence, it might use other
+  # key names than Gardener itself.
+  # The actual values here depend on the DNS extension that is installed to your landscape.
+  # For example, check out https://github.com/gardener/external-dns-management and find a lot of example secret manifests here:
+  # https://github.com/gardener/external-dns-management/tree/master/examples
+
+---
+apiVersion: core.gardener.cloud/v1alpha1
+kind: SecretBinding
+metadata:
+  name: secretbinding1
+  namespace: garden-core
+secretRef:
+  name: secret1
+# namespace: namespace-other-than-'garden-core' // optional
+quotas: []
+# - name: quota-1
+# # namespace: namespace-other-than-'garden-core' // optional
+```
+
+### `Quota` resource
+
+Special note: No modifications needed compared to the current `garden.sapcloud.io/v1beta1.Quota` resource.
+
+```yaml
+apiVersion: core.gardener.cloud/v1alpha1
+kind: Quota
+metadata:
+  name: trial-quota
+  namespace: garden-trial
+spec:
+  scope:
+    apiGroup: core.gardener.cloud
+    kind: Project
+# clusterLifetimeDays: 14
+  metrics:
+    cpu: "200"
+    gpu: "20"
+    memory: 4000Gi
+    storage.standard: 8000Gi
+    storage.premium: 2000Gi
+    loadbalancer: "100"
+```
+
+### `BackupBucket` resource
+
+Special note: This new resource is cluster-scoped.
```

Comment:
It might be better to rename this resource to `ClusterBackupBucket` to avoid confusion.

Reply:
@swapnilgm any thoughts?
I have no strong opinion on this, IMHO it makes only sense to prefix it with `Cluster` if we plan to introduce a namespaced `BackupBucket`. I don't think we do.

Reply:
Agreed with @rfranzke. We don't have any plan to introduce the namespaced BackupBucket. So, lets keep it as it is.